users/peej/peej.h

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2020 Paul James
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+#pragma once
+
+#include "eeconfig.h"
+
+#ifdef RGBLIGHT_ENABLE
+    #include "rgblight.h"
+#endif
+
+#define PERMISSIVE_HOLD
+#define TAPPING_TERM 100
+
+#define _BASE 0
+#define _SYMBOL 3
+#define _FUNCTION 4
+#define _ADJUST 16
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define ______2 KC_TRNS, KC_TRNS
+#define ______3 KC_TRNS, KC_TRNS, KC_TRNS
+#define ______4 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+#define ______5 KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```